index.html

```diff
@@ -724,14 +705,8 @@ <h4>Roles which cannot be named (Name prohibited)</h4>
             </section>
 		</section>
 		<section id="childrenArePresentational">
-			<h3>Presentational Children</h3>
-			<dl class="runin">
-				<dt>Values</dt>
-				<dd>
-					<p>Boolean (<code>true</code> | <code>false</code>)</p>
-				</dd>
-			</dl>
-			<p>The <abbr title="Document Object Model">DOM</abbr> descendants are presentational. <a>User agents</a> SHOULD NOT expose descendants of this <a>element</a> through the platform <a>accessibility <abbr title="Application Programing Interface">API</abbr></a>. If <a>user agents</a> do not hide the descendant nodes, some information may be read twice.</p>
+			<h3>Children Presentational</h3>
+			<p>Indicates whether owned elements are presentational. <a>User agents</a> SHOULD NOT expose descendants of this <a>element</a> through the platform <a>accessibility <abbr title="Application Programing Interface">API</abbr></a>. If <a>user agents</a> do not hide the descendant nodes, some information may be read twice.</p>
```

Comment:
Is owned elements the same as DOM descendents here? owned would include aria-owns. Does Children Presentational cascade to that? (asking the question - I'm not sure of the answer). It would seem that aria-owns should probablty be prohibited on anything with children presentational true but we don't currently specify that....

Reply:
Sorry for missing this last month.

> Is owned elements the same as DOM descendents here? 

Good question. I must have been overeager.

>  Does Children Presentational cascade to that? (asking the question - I'm not sure of the answer). 

Can we add this question to the agenda?

>  It would seem that aria-owns should probablty be prohibited on anything with children presentational true but we don't currently specify that...

Makes sense to me.

Reply:
we will tackle aria-owns restrictions in https://github.com/w3c/aria/issues/1760